### ISSUE TITLE
Cron backup regex format

### ISSUE THREAD

**Issue by LEI** [Promaethius/mailcow-dind#9]
When attempting to deploy the example config on kubernetes, the mailcow-dind pod fails to start with this error message: `Incorrect format for 0 0 * * 0`.

While it is a [valid cron expression](https://crontab.guru/#0_0_*_*_0), CRON_REGEX seems to have an extra capture group in ./images/mailcow-dind/cmd.sh:

```
CRON_REGEX='/^([0-59]|\*) ([0-23]|\*) ([1-31]|\*) ([1-12]|\*) ([1-7]|\*) ([1900-3000]|\*)$/'
```

If the last part should be omitted, the CRON_BACKUP value (currently `* * * * * *`) should reflect this change in order to run the container from .travis.yml.

**Comment by Promaethius** [comment]
@LEI thanks for pointing that out. The package `crond` from alpine allows an entry for years which I added but totally forgot to represent that in the example. Let me know if the change works out for you.

**Comment by LEI** [comment]
It seems mailcow does not like this format, it still enters in a crash loop:

```
2018-12-05 21:14:42,535 INFO Set uid to user 0 succeeded
2018-12-05 21:14:42,539 INFO supervisord started with pid 6
2018-12-05 21:14:43,543 INFO spawned: 'cmd' with pid 9
2018-12-05 21:14:43,546 INFO spawned: 'cron' with pid 10
2018-12-05 21:14:43,555 INFO spawned: 'docker' with pid 11
/usr/sbin/crond 4.5 dillon's cron daemon, started with loglevel notice
CRON_BACKUP must be set in cron format for consistent backups.
2018-12-05 21:14:44,742 INFO success: cmd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2018-12-05 21:14:44,742 INFO success: cron entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2018-12-05 21:14:44,742 INFO success: docker entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2018-12-05 21:14:53,587 INFO exited: cmd (exit status 1; not expected)
2018-12-05 21:14:54,595 INFO spawned: 'cmd' with pid 183
CRON_BACKUP must be set in cron format for consistent backups.
2018-12-05 21:14:55,630 INFO success: cmd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2018-12-05 21:15:04,629 INFO exited: cmd (exit status 1; not expected)
2018-12-05 21:15:05,633 INFO spawned: 'cmd' with pid 189
CRON_BACKUP must be set in cron format for consistent backups.
2018-12-05 21:15:06,687 INFO success: cmd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2018-12-05 21:15:15,665 INFO exited: cmd (exit status 1; not expected)
2018-12-05 21:15:16,670 INFO spawned: 'cmd' with pid 193
CRON_BACKUP must be set in cron format for consistent backups.
2018-12-05 21:15:17,722 INFO success: cmd entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2018-12-05 21:15:26,728 INFO exited: cmd (exit status 1; not expected)
```

**Comment by Promaethius** [comment]
That's probably more of an issue with my tagging (the latest tag was over 2 months old and likely had a bug). I'll work on a better travis deployment strategy later today; but, in the meantime, I've modified the kubernetes example to use the docker image master tag.

**Comment by LEI** [comment]
Ignore the previous log, CRON_BACKUP was not set, my bad. However I still encountered `Incorrect format for 0 0 * * 0 *`, but I found out the week part of the expression must have a value between 0 and 6, at least in a "standard" cron : replacing `([1-7]|\*)` with `([0-6]|\*)` seems to work.

Other than that I had to tweak 2 or 3 things in the YAML files for it to work with my version of kubernetes, like lowercasing the image value in `deployment.yaml` and the port names in `service.yaml`.
